### ISSUE TITLE
The performance degradation may be caused by overfitting in the distillation stage

### ISSUE THREAD

**Issue by tju-xuan** [lgz-0713/ReDDiT#10]
Dear authors,

I encountered an issue while training with the default ReDDiT script. As shown in the log below:

<img width="1079" height="324" alt="Image" src="https://github.com/user-attachments/assets/8db33ac3-e5ed-41ae-a890-b2bf2c3f2af1" />

All evaluation metrics (PSNR, SSIM, and LPIPS) consistently degrade over the course of training.

This seems abnormal and possibly indicates an issue such as overfitting during the distillation process.

Could you please confirm whether the default hyperparameters in the provided code are correct?

If possible, could you also share the training configuration (e.g., learning rate schedule, loss weights, etc.) you used in your experiments?

Thank you in advance!

**Comment by tju-xuan** [comment]
btw, I found that setting w_snr = 0 increases the PSNR, which suggests that the RATR module is not working as intended.

**Comment by 1809135** [comment]
I have also discovered this issue and hope to receive a response.Thank you!

**Comment by lgz-0713** [comment]
Thank you for your interest in our research. Since the training dataset is relatively small, models tend to overfit easily. Our method uses early stopping to mitigate overfitting. This means that the released distilled model has already been fully optimized and may not be further fine-tuned.

To reproduce:

1. Train a vanilla diffusion model with diffusion loss only.

2. Run our distillation on that checkpoint and use early stopping.
